src/main/cpp/DragonVision/DragonLimelight.cpp

```diff
@@ -454,10 +463,16 @@ std::optional<units::length::inch_t> DragonLimelight::EstimateTargetZDistance()
 
 std::optional<units::length::inch_t> DragonLimelight::EstimateTargetXDistance_RelToRobotCoords()
 {
-    if (EstimateTargetXDistance())
+    std::optional<units::length::inch_t> optionalTargetXDistance = EstimateTargetXDistance();
+    if (optionalTargetXDistance)
     {
-        units::length::inch_t targetXoffset_RF_inch = EstimateTargetXDistance().value() + GetMountingXOffset(); ///< the offset is negative if the limelight is behind the center of the robot
-
+        units::length::inch_t targetXoffset_RF_inch{0.0};
+        try {
+            targetXoffset_RF_inch = optionalTargetXDistance.value() + GetMountingXOffset(); ///< the offset is negative if the limelight is behind the center of the robot
+            //Logger::GetLogger()->LogData(LOGGER_LEVEL::PRINT, std::string("LL"), std::string("EstimateTargetXDistance_RelToRobotCoords"), targetXoffset_RF_inch.to<double>());
+        } catch (std::bad_optional_access &e) {
+            //Logger::GetLogger()->LogData(LOGGER_LEVEL::PRINT, std::string("LL"), std::string("EstimateTargetXDistance_RelToRobotCoords-exception"), e.what());
```

Comment:
If we are going to keep this, can we return std::nullopt when we catch an exception

Reply:
i removed that as I was able to track it down